VMEncryption/main/Common.py

```diff
@@ -94,6 +94,7 @@ class CommonVariables:
     default_mount_name = 'encrypted_disk'
     dev_mapper_root = '/dev/mapper/'
     osmapper_name = 'osencrypt'
+    cvmosmapper_name = 'cloudimg-rootfs'
```

Comment:
remove cvmosmapper_name, as this is not used in code. 
using lablk to find device information.

Reply:
removed